perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -387,8 +418,8 @@ def WriteObjectFromBuffer(storage_schema, bucket_name, object_name,
     storage_schema: The address schema identifying a storage. e.g., "gs"
     bucket_name: Name of the bucket to write to.
     object_name: Name of the object.
-    payload_bytes: A bytearray to transfer.
-    payload_string: A string version of payload_bytes.
+    stream: A read()-able stream to transfer.
```

Comment:
Consider adding a condition/constraint here: 

"This function will always seek to the beginning of the stream and write the requested number of bytes from that position to the object storage"

This way, the caller does not have to shoulder the burden of doing seek(0) every time then call this function. (and currently, we don't have a use case to write from a middle of the stream).